src/ppx_deriving.cppo.ml

```diff
@@ -12,6 +12,21 @@ open Parsetree
 open Ast_helper
 open Ast_convenience
 
+#if OCAML_VERSION >= (4, 05, 0)
+module Typ = struct
+  include Typ
+
+  let poly ?(loc= !Ast_helper.default_loc) ?attrs vars ty =
+    let vars = List.map (fun txt -> { Asttypes.loc; txt }) vars in
+    Typ.poly ~loc ?attrs vars ty
```

Comment:
Do you think something should be changed in this PR? I didn't change the API of `free_vars_in_core_type` to avoid breaking reverse dependencies. I suppose we could add an alternative function returning located strings and deprecate `free_vars_in_core_type`